### ISSUE TITLE
module 'lodash' not found v0.8.1

### ISSUE THREAD

**Issue by EthanRubinson** [fergiemcdowall/search-index#270]
It looks like the 'lodash' module was removed from the package.json file. I repeatedly get the following stack-trace when trying to add something to an index:

```
 Uncaught Error: Cannot find module 'lodash'
      at require (internal/module.js:12:17)
      at module.exports (node_modules/search-index/lib/siUtil.js:5:11)
      at node_modules/search-index/lib/index.js:39:44
      at node_modules/search-index/node_modules/async/lib/async.js:726:13
      at node_modules/search-index/node_modules/async/lib/async.js:52:16
      at node_modules/search-index/node_modules/async/lib/async.js:269:32
      at node_modules/search-index/node_modules/async/lib/async.js:44:16
      at node_modules/search-index/node_modules/async/lib/async.js:723:17
      at node_modules/search-index/node_modules/async/lib/async.js:167:37
      at node_modules/search-index/node_modules/search-index-searcher/searcher.js:83:12
      at node_modules/search-index/node_modules/search-index-searcher/searcher.js:669:12
      at node_modules/search-index/node_modules/async/lib/async.js:726:13
      at node_modules/search-index/node_modules/async/lib/async.js:52:16
      at async.forEachOf.async.eachOf (node_modules/search-index/node_modules/async/lib/async.js:236:30)
      at _parallel (node_modules/search-index/node_modules/async/lib/async.js:717:9)
      at Object.async.parallel (node_modules/search-index/node_modules/async/lib/async.js:731:9)
      at getOptions (node_modules/search-index/node_modules/search-index-searcher/searcher.js:635:9)
      at module.exports (node_modules/search-index/node_modules/search-index-searcher/searcher.js:35:3)
      at async.series.searchIndexAdder (node_modules/search-index/lib/index.js:29:41)
      at node_modules/search-index/node_modules/async/lib/async.js:718:13
      at Immediate.iterate [as _onImmediate] (node_modules/search-index/node_modules/async/lib/async.js:262:13)
```

Running `npm install lodash --save` in the nodeModules/search-index directory resolved it.

**Comment by fergiemcdowall** [comment]
Ahh- oops!

Lodash was removed from the lib in order to make the browser build smaller, I see now that there was a `require _ = require('lodash')`still hanging around there. (Aside: If you need a lib in `devDependencies`, but not `dependencies`how do you test for its exclusion?)

Should be fixed in 0.8.2 which is out on npm now

**Comment by fergiemcdowall** [comment]
As far as I can see, this is fixed- please reopen if the problem persists
